src/wymeditor/core.js

```diff
@@ -474,6 +474,63 @@ jQuery.extend(WYMeditor, {
         TEXT: 3
     },
 
+    // These UI components will preferentially use those defined in the chosen
+    // skin inside `UI_COMPONENTS` instead of the defaults.
```

Comment:
I feel that a big readability improvement can be easily gained by replacing "those" with a something more descriptive.